tox.ini

```diff
@@ -11,25 +11,15 @@ passenv =
     AWS_SECRET_ACCESS_KEY
     AWS_SESSION_TOKEN
 usedevelop = true
+deps =
+    redis
+    coverage
```

Comment:
Removing the `!nodeps` factorized installation of `boto3` and `moto` means that tests requiring boto3+moto will never run.